src/topology/maps.lean

```diff
@@ -342,6 +342,33 @@ lemma to_quotient_map {f : α → β}
 
 end is_open_map
 
+section is_closed_map
+variables [topological_space α] [topological_space β]
+
+def is_closed_map (f : α → β) := ∀ U : set α, is_closed U → is_closed (f '' U)
+
+end is_closed_map
+
+namespace is_closed_map
+
+variables [topological_space α] [topological_space β] [topological_space γ]
+open function
+
+protected lemma id : is_open_map (@id α) := assume s hs, by rwa image_id
```

Comment:
is_closed_map ?

Reply:
Thanks, good catch.

Reply:
And the proof doesn't change, I love that!